docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,163 @@
+        when size based split weights are enabled.
+      - ``128MB``
+    * - ``hudi.minimum-assigned-split-weight``
+      - Minimum weight that a split can be assigned
+        when size based split weights are enabled.
+      - ``0.05``
+    * - ``hudi.max-splits-per-second``
+      - Rate at which splits are enqueued for processing.
+        The queue will throttle if this rate limit is breached.
+      - ``Integer.MAX_VALUE``
+    * - ``hudi.max-outstanding-splits``
+      - Maximum outstanding splits in a batch enqueued for processing.
+      - ``1000``
+
+Supported file types
+--------------------
+
+The following file types are supported for the Hudi connector:
+
+* Parquet
+
+SQL Support
+-----------
+
+Currently, the connector only provides read access to data in the Hudi table that has been synced to
+Hive metastore. Once the catalog has been configured as mentioned above, users can query the tables
+as usual like Hive tables.
+
+Supported Query Types
+^^^^^^^^^^^^^^^^^^^^^
+
+Hudi supports two types of tables depending on how the data is indexed and laid out on
+the file system. To learn more about table and query types in Hudi, please check
+`this doc <https://hudi.apache.org/docs/table_types>`_. Below table is the support matrix
+for Hudi connector.
+
+=========================== =============================================
+Table Type                  Supported Query types
+=========================== =============================================
+Copy On Write               Snapshot Queries
+
+Merge On Read               Read Optimized Queries
+=========================== =============================================
+
+Examples Queries
+^^^^^^^^^^^^^^^^
+
+``stock_ticks_cow`` is a Hudi COW table that we refer in the Hudi
+`quickstart <https://hudi.apache.org/docs/docker_demo/>`_ document to create.
+
+Here are some sample queries:
+
+.. code-block:: sql
+
+    USE hudi.default;
+    select symbol, max(ts) from stock_ticks_cow group by symbol HAVING symbol = 'GOOG';
+
+.. code-block:: text
+
+      symbol   |        _col1         |
+    -----------+----------------------+
+     GOOG      | 2018-08-31 10:59:00  |
+    (1 rows)
+
+.. code-block:: sql
+
+    select dt, symbol from stock_ticks_cow where symbol = 'GOOG';
+
+.. code-block:: text
+
+        dt      | symbol |
+    ------------+--------+
+     2018-08-31 |  GOOG  |
+    (1 rows)
+
+.. code-block:: sql
+
+    select dt, count(*) from stock_ticks_cow group by dt;
+.. code-block:: text
+
+        dt      | _col1 |
+    ------------+--------+
+     2018-08-31 |  99  |
+    (1 rows)
+
+Reading Hudi Table using the Hive Connector
+-------------------------------------------
+
+Hudi tables can also be queried using the Hive connector. The supported query types with
+Hive connector are same as that of Hudi connector. To query Hudi tables on Trino, please
+place the `hudi-trino-bundle <https://mvnrepository.com/artifact/org.apache.hudi/hudi-trino-bundle>`_
```

Comment:
Dont link to mvnrepository .. instead use the official maven repo or the official maven repo search  

https://repo.maven.apache.org/


https://central.sonatype.dev/